### PR TITLE
Put a copyright header on new files from IDEA

### PR DESCRIPTION
**Pull request by JakeWharton**

### CODE REVIEW

File: .idea/copyright/Square.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="notice" value="Copyright (C) &amp;#36;{today.year} Square, Inc.&#10;&#10;Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);&#10;you may not use this file except in compliance with the License.&#10;You may obtain a copy of the License at&#10;&#10;     http://www.apache.org/licenses/LICENSE-2.0&#10;&#10;Unless required by applicable law or agreed to in writing, software&#10;distributed under the License is distributed on an &quot;AS IS&quot; BASIS,&#10;WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.&#10;See the License for the specific language governing permissions and&#10;limitations under the License." />
```
Comment (adrw) [inline .idea/copyright/Square.xml]: Is this supposed to be Block?

**Review by adrw — APPROVED** [review]

File: .idea/copyright/Square.xml (hunk 1 above)
Comment (JakeWharton) [reply .idea/copyright/Square.xml]: Ew, hell no! I can do "Cash App" if that's what we want. Lawyers okayed that, too.

**Comment by JakeWharton** [comment]
Note that once we get Spotless I think we'll have to normalize them all to be the same… we'll see.

File: .idea/copyright/Square.xml (hunk 1 above)
Comment (adrw) [reply .idea/copyright/Square.xml]: Cash App would be dope

**Review by swankjesse — COMMENTED** [review]
Nice

**Pull request merged**